.github/workflows/ci.yml

```diff
@@ -47,44 +47,7 @@ jobs:
         run: |
           nvm use
           npm run storybook:dist
-
-      - name: Run visual tests
-        id: visualTests
-        run: |
-          nvm use
-          npm run jest:visual-tests
-
-      - name: Upload failed visual tests (if necessary)
-        uses: actions/upload-artifact@v2
-        if: failure() && steps.visualTests.outcome == 'failure'
-        with:
-          name: failed-visual-tests
-          path: .storybook/__image_snapshots__/__diff_output__
-
-      - name: Post a comment about the visual tests (if they pass)
-        uses: actions/github-script@v5
-        if: github.ref != 'refs/heads/main' && steps.visualTests.outcome == 'success' && github.repository == github.event.pull_request.head.repo.full_name  && github.actor != 'dependabot[bot]' 
-        with:
-          script: |
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: "Visual regression tests passed 😎. Bear in mind that they only run in Chromium on static components – they aren't perfect."
-            })
-
-      - name: Post a comment about the visual tests (if they fail)
-        uses: actions/github-script@v5
-        if: failure() && steps.visualTests.outcome == 'failure' && github.repository == github.event.pull_request.head.repo.full_name && github.actor != 'dependabot[bot]'
-        with:
-          script: |
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: "Visual regression tests failed 😢. You can download the failure diffs from the 'Artifacts' section of the failed CI run. To update the tests, run `npm run jest:visual-tests:update` locally."
-            })
-
```

Comment:
Can we just comment this instead of removing so its easier to reactive in the future?